index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<!--Created HTML document-->
+
+<body>on paper<br><br>
+    by Jacqueline Woodson
+<br><br>
+    The first time I write my full name
+<br>
+    Jacqueline Amanda Woodson
+<br>
+    without anybody&apos;s help
+    on a clean white page in my composition notebook,<br>
+    I know<br>
+
+    if I wanted to<br>
+
+    I could write anything.<br>
+
+    Letters becoming words, words gathering meaning,
+    becoming<br>
+    thoughts outside my head
+
+    becoming sentences<br><br>
+
+    written by<br><br>
+
+    Jacqueline Amanda Woodson
+
+</body> 
```

Comment:
I would definitely expect to see additional markup around the text in this document. You may find it helpful to review the HTML readings on:
- [headings](https://mica-web.github.io/learn/html/headings)
- [text](https://mica-web.github.io/learn/html/text)

At the very least, this HTML should have a:
- `h1` for the title/main topic of this content
- `h2` or `p` surrounding the author's byline (line 12)
- `p` surrounding one or more of the bits of the poem's text

Because your HTML is missing these elements, you've had to force spacing by using a double line break (`<br><br>`). While `br` is appropriate for the text of the poem (in order to maintain the line breaks the author created), they are not appropriate for separating the poem's title from the author's name, for example (see line 11).

Let me know if you want to revisit this yourself or if an example would be most helpful to you?